R/run_stress_test.R

```diff
@@ -215,7 +210,7 @@ read_and_process <- function(args_list) {
     process_company_terms(fallback_term = term)
 
   cat("-- Processing input data. \n")
-  processed <- st_process(data, asset_type)
+  processed <- st_process(data, asset_type, company_exclusion)
```

Comment:
number of arguments is starting to grow, how about using names in the call to st_process?

Reply:
Done